chapter1/element.md

```diff
@@ -159,58 +164,61 @@ defmodule Your.Module.Element do
       counter: 0,
       timer: nil
     }
+
     {:ok, state}
   end
 
   @impl true
-  def handle_play(state) do
+  def handle_prepared_to_stopped(_ctx, state) do
+    {:ok, :cancel} = :timer.cancel(state.timer)
+    {:ok, %{state | counter: 0, timer: nil}}
+  end
+
+  @impl true
+  def handle_prepared_to_playing(_ctx, state) do
     {:ok, timer} = :timer.send_interval(state.interval, :tick)
     {:ok, %{state | timer: timer}}
   end
 
   @impl true
-  def handle_demand :source, size, :bytes, _context, state do
-    {{:ok, [demand: {:sink, size}]}, state}
+  def handle_demand(:output, size, :bytes, _context, state) do
+    {{:ok, [demand: {:input, size}]}, state}
   end
 
   @impl true
-  def handle_process1 :sink, %Membrane.Buffer{} = buffer, _, state do
+  def handle_process(:input, %Membrane.Buffer{} = buffer, _, state) do
     new_state = %{state | counter: state.counter + 1}
-    {{:ok, [buffer: {:source, buffer}]}, new_state}
+    {{:ok, [buffer: {:output, buffer}]}, new_state}
   end
 
   @impl true
-  def handle_other(:tick, state) do
-    # create message to send
-    message = %Membrane.Message{
-      type: :statistics,
-      payload: %{counter: state.counter}
+  def handle_other(:tick, _ctx, state) do
+    # create term to send
+    notification = {
+      :counter,
+      state.counter
     }
 
     # reset the timer
     new_state = %{state | counter: 0}
 
-    {{:ok, [message: message]}, new_state}
-  end
-
-  @impl true
-  def handle_stop(state) do
-    {:ok, :cancel} = :timer.cancel(state.timer)
-    {:ok, %{state | counter: 0,  timer: nil}}
+    {{:ok, [notify: notification]}, new_state}
   end
 end
 ```
 
 ## Test the element
 
-Our element is now ready! The last step is to put it in some pipeline and add callback handling messages in the pipeline. The simple example of such callback is the following:
+Our element is now ready! The last step is to put it in some pipeline and add callback handling notifications in the pipeline. The simple example of such callback is the following:
 
 ```elixir
 @impl true
-def handle_message(message, _elem_name, state) do
-  IO.inspect message
+def handle_notification(notification, _elem_name, state) do
+  IO.inspect(notification)
   {:ok, state}
 end
 ```
 
 You can use the pipeline from the previous chapter and put this element between the sink and the decoder.
+
+You can clone the complete "first element" demo from [here](https://github.com/membraneframework/membrane-demo/tree/v0.2).
```

Comment:
From what I've seen, usually, such links are placed at the beginning of the tutorial